tests/unit/test_controlTypes.py

```diff
@@ -16,7 +16,7 @@ class TestLabels(unittest.TestCase):
 	def test_roleLabels(self):
 		"""Test to check whether every role has its own label in controlTypes.roleLabels"""
 		for name, const in vars(controlTypes).items():
-			if name.startswith("ROLE_"):
+			if name.startswith("Role."):

```

Comment:
this unit test no longer tests anything. In the Python console:
```
>>> import controlTypes
>>> for name, const in vars(controlTypes).items():
... 	if name.startswith("Role."):
... 		print(name)
... 

prints nothing.

If you want to just modify it to test enum values you should iterate through all its members, however since we want to ensure backwards compatibility ideally this unit test should be converted into two. First should test an enum values as described above, and the second one should be an exact copy of the one on current master but decorated with `@unittest.skipIf()` with a condition ensuring that  it would not run when `buildVersion.version_year >= 2022`

Reply:
Thanks for picking up on this, I've implemented a new test